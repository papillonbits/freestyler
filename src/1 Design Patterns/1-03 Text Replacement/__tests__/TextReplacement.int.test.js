import { regular } from './TextReplacement.int.story'

describe('<TextReplacement />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
