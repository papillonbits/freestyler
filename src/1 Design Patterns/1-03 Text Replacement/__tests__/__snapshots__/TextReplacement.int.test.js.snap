// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextReplacement /> Render must match regular() 1`] = `
<div>
  <div
    class="text-replacement"
  >
    <h1>
      Text Replacement
    </h1>
    <h2
      id="h2"
    >
      Heading 2
      <span />
    </h2>
  </div>
</div>
`;
